components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -112,14 +128,32 @@ const HighlightInputForm = (): JSX.Element => {
           ref={textAreaRef}
           rows={row}
           value={bodyText}
-          onChange={(e) => handleTextAreaInputChange(e)}
+          onChange={(e) => {
+            handleTextAreaInputChange(e);
+            setWordCount(e.target.value.length);
+          }}
         />
+        {isDivFocused && (
+          <p className="text-xs p-2 text-light-slate-9 flex justify-end gap-1">
+            <span className={`${wordCount - pullrequestLink.length > wordLimit && "text-red-600"}`}>
```

Comment:
This logic is repeated. It would be best to put the charCount in a reusable function or service.

Also this is not a word count but a character count.